code/datums/status_effects/debuffs.dm

```diff
@@ -824,3 +824,92 @@
 
 /datum/status_effect/transient/drugged/on_remove()
 	owner.update_druggy_effects()
+
+#define FAKE_COLD 1
+#define FAKE_FOOD_POISONING 2
+#define FAKE_RETRO_VIRUS 3
+#define FAKE_TURBERCULOSIS 4
+
+/datum/status_effect/fake_virus
+	id = "fake_virus"
+	duration = 3 MINUTES
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 2
+	alert_type = null
+	/// So you dont get the most intense messages immediately
+	var/msg_stage = 0
+	/// Which disease we are going to fake?
+	var/current_fake_disease
+	// Fake virus messages by stage
+	var/list/fake_msg_1
+	var/list/fake_msg_2
+	var/list/fake_msg_3
+	// Fake virus emotes by stage
+	var/list/fake_emote_1
+	var/list/fake_emote_2
+	var/list/fake_emote_3
```

Comment:
I'm not that keen on having 2 sets of 3 lists. Could you make 2 lists with 3 lists in them instead? Then use an index you determine by the `msg_stage` variable to access them?
Should clean up the code in `tick` as well

Reply:
Like this?